tools/celcli/go.mod

```diff
@@ -0,0 +1,13 @@
+module celcli
```

Comment:
Let's change the module name to the full path `github.com/kubescape/cel-admission-library`